swagger_spec_validator/schemas/v2.0/json_schema_draft04.json

```diff
@@ -1,7 +1,7 @@
 {
-    "id": "custom_job_schema.json#",
-    "$schema": "custom_job_schema.json#",
-    "description": "Custom schema meta-schema that allows for additional simple types",
+    "id": "http://json-schema.org/draft-04/schema#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Just checking that these URLs don't actually get used during validation.

Reply:
Page still worked, and tests passed but. It was adding a warning onto the page. I went ahead and removed the urls and used the file name instead. So i guess our schema is almost the same as their file for the json stuff